src/component/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { fetchIndexMessage } from "../redux/action/IndexAction";
+import { IndexState } from "../redux/store";
+
+function InitialMessage() {
+    const indexData = useSelector((state: IndexState) => state.index.data)
+    const indexStatus = useSelector((state: IndexState) => state.index.status)
+    const indexLoading = useSelector((state: IndexState) => state.index.loading)
+    const indexError = useSelector((state: IndexState) => state.index.error)  
+//   const index = useSelector(
```

Comment:
why is this commented

Reply:
They are not necessary, I've removed them now.